inst/vign/taxize_case_study.Rmd

```diff
@@ -0,0 +1,68 @@
+<!--
+%\VignetteEngine{knitr::knitr}
+%\VignetteIndexEntry{Strategies for programmatic name cleaning}
```

Comment:
seems the title here  should be `Case study - resolving Species Plantarum names`

Reply:
Thanks for the comments, I'll take a look at the weekend hopefully. Very much a work in progress, happy to change things, just point me in the right direction.

Reply:
sounds good